gapis/api/gles/gles.api

```diff
@@ -58,11 +58,9 @@ class Versions {
   bool GLES31 = true
   bool GLES32 = true
 }
-// TODO: This should be thread local variable
-Versions Version
 
-sub void InitVersions(ref!Context ctx) {
-  Version = Versions(
+sub ref!Versions InitVersions(ref!Context ctx) {
+  return new!Versions(
     GLES10:  VersionGreaterOrEqual(ctx, 1, 0),
```

Comment:
I'd really like it if these were called `AtLeastGLESXX`. The way I read them right now is an exact version match.

Reply:
I will just change the name of the struct to supported versions.
NB: Mind that the primary purpose of this is to be able to add the @if(Version.GLES30) annotation.